### DISCUSSION TITLE
0.10.13

### DISCUSSION THREAD

**Discussion by danwalmsley** [AvaloniaUI/Avalonia#7737] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/AvaloniaUI/Avalonia/releases/tag/0.10.13'>0.10.13</a>.</em>

**Comment by flarive** [comment]
Hi !
Where can i find the changelog for 0.10.13 new version please ?

Thanks a lot !

**Reply by robloo** [reply]
GitHub can do an automatic summary of the release commits https://github.com/AvaloniaUI/Avalonia/compare/0.10.12...0.10.13

**Reply by Takoooooo** [reply]
changelog is available now :) sorry for delay

**Reply by flarive** [reply]
Thanks !
